sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -44,12 +47,28 @@ fun HomeScreen(
     )
 }
 
+@OptIn(ExperimentalPermissionsApi::class)
 @Composable
 private fun HomeScreenContent(
     uiModels: List<UiModel>,
     showLoading: IsLoading,
     onItemClick: (UiModel) -> Unit
 ) {
+    val cameraPermissionState = rememberPermissionState(CAMERA)
+    if (cameraPermissionState.status.isGranted) {
+        Timber.d("${cameraPermissionState.permission} granted")
+    } else {
+        if (cameraPermissionState.status.shouldShowRationale) {
+            Timber.d("${cameraPermissionState.permission} needs rationale")
+        } else {
+            Timber.d("Request cancelled, missing permissions in manifest or denied permanently")
+        }
+
+        LaunchedEffect(Unit) {
```

Comment:
Let's define the key instead of `Unit`
```suggestion
        LaunchedEffect(cameraPermissionState) {
```

Reply:
@Wadeewee Nope, we only need to execute it 1 time each state changes. It will be re-executed when `cameraPermissionState` changes.

Reply:
Ah, I see because It's state, So, That makes sense to use `Unit` 👍